storageapi/src/vespa/storageapi/messageapi/returncode.h

```diff
@@ -121,8 +102,7 @@ class ReturnCode : public document::Deserializable,
     bool isShutdownRelated() const;
     bool isBucketDisappearance() const;
     bool isNonCriticalForIntegrityChecker() const;
+    vespalib::string toString() const;
```

Comment:
Consider adding explicit `operator<<` for `ReturnCode`; it is used in many unit test macros so having it ostreamable makes "expectation vs. reality" printing much better.

This would also remove need to add `toString()` calls in a lot of the result printing functions.

Reply:
Done